### DISCUSSION TITLE
error in tutorial 4

### DISCUSSION THREAD

**Discussion by YoungAgile** [pybamm-team/PyBaMM#3154] [General]
https://docs.pybamm.org/en/latest/source/examples/notebooks/getting_started/tutorial-4-setting-parameter-values.html
In this tutorial, when i run those code, it return keyerror : 'Cannot update parameter 'Open-circuit voltage at 100% SOC [V]' as it does not have a default value. ' 
```
parameter_values["Current function [A]"] = 10
parameter_values["Open-circuit voltage at 100% SOC [V]"]=3.4
parameter_values["Open-circuit voltage at 0% SOC [V]"]=3.0
```

Should I use `'Upper voltage cut-off [V]'`and` 'Lower voltage cut-off [V]'` to replace them?

**Comment by brosaplanella** [comment]
Try doing
```python3
parameter_values.update(
    {
        "Open-circuit voltage at 100% SOC [V]": 3.4,
        "Open-circuit voltage at 0% SOC [V]": 3.0,
    },
    check_already_exists=False
)
```

**Reply by YoungAgile** [reply]
Thank you! And should the code in the tutorial be updated

**Reply by brosaplanella** [reply]
This should be automatically fixed in the new release
